src/components/editPostModal/index.js

```diff
@@ -2,19 +2,27 @@ import { useState } from "react"
 import useModal from "../../hooks/useModal"
 import './style.css'
 import Button from '../button'
+import { updatePost } from "../../service/apiClient"
 
-const EditPostModal = () => {
+const EditPostModal = ({content, id, setContent}) => {
     const { closeModal } = useModal()
     const [message, setMessage] = useState(null)
-    const [text, setText] = useState('')
+    const [text, setText] = useState(content)
+
 
     const onChange = (e) => {
         setText(e.target.value)
     }
 
-    const onSubmit = () => {
+    async function onSubmit ()  {
         setMessage('Submit button was clicked! Closing modal in 2 seconds...')
-
+        let test = await updatePost(id, text)
+        if(test.status == "fail"){
+            console.log(test.message);
```

Comment:
Let's get rid of any console logs, you don't want debug logs to make it into the main branch